contactql/parser_test.go

```diff
@@ -86,7 +88,7 @@ func TestParseQuery(t *testing.T) {
 
 		// implicit combinations
 		{`will felix`, `name ~ "will" AND name ~ "felix"`, "", envs.RedactionPolicyNone},
-		{`will +123456566`, `name ~ "will" AND tel ~ 123456566`, "", envs.RedactionPolicyNone},
+		{`will +123456566`, `name ~ "will" AND tel ~ +123456566`, "", envs.RedactionPolicyNone},
```

Comment:
Feels slightly weird to not have the number quoted when it leads with a `+` but I guess I'm fine with it.

Reply:
Was thinking same thing - would be easy to tweak that

Reply:
Updated